doc/stdlib/linalg.rst

```diff
@@ -101,6 +101,10 @@ Linear algebra functions in Julia are largely implemented by calling functions f
 
    Compute a factorization of a positive definite matrix ``A`` such that ``A=L*Diagonal(d)*L'`` where ``L`` is a unit lower triangular matrix and ``d`` is a vector with non-negative elements.
 
+.. function:: ldltfact(A) -> CHOLMOD.Factor
+
+   Compute the LDLt factorization of a sparse symmetric or Hermitian matrix ``A``. A fill-reducing permutation is used.  The main application of this type is to solve systems of equations with ``\``, but also the methods ``diag``, ``det``, ``logdet`` are defined. The function calls the C library CHOLMOD and many other functions from the library are wrapped but not exported.
```

Comment:
This is either inaccurately documented or broken for indefinite matrices where diagonal-D factorization fails. I've said it before and I'll continue saying it until we get it right, general symmetric indefinite matrices require factorizations where D is either block-diagonal, or tridiagonal. The fact that cholmod can't provide this functionality is not an excuse for our documentation to be misleading.

```
julia> A = sprand(500,500,0.4); B = A + A';
julia> diag(ldltfact(B))
500-element Array{Float64,1}:
 0.0
 0.0
 0.0
...
```